spec/requests/api/v1/moves_controller_update_spec.rb

```diff
@@ -496,6 +556,42 @@
               expect(move.reload.allocation).to be_unfilled
             end
           end
+
+          context 'when linking a profile' do
```

Comment:
Not sure about having those tests here as well as in the updater spec, but added them to mirror the person tests for allocation already defined. Maybe we should remove both 🤔

Reply:
I would remove them both. Happy if you want to make it part of this PR

Reply:
By both, I mean the duplication in the request spec